api/docs/docs/data-definitions.md

```diff
@@ -146,21 +146,21 @@ Fields:
 
 ### Beds With Covid Patients Ratio
 
-  Ratio of staffed hospital beds that are currently in use by COVID patients. For counties, this is calculated using health service area data for the corresponding area. For more information on HSAs see the [Health Service Area section](https://apidocs.covidactnow.org/data-definitions/#health-service-areas)
+  Ratio of staffed hospital beds that are currently in use by COVID patients. For counties, this is calculated using HSA-level data for the corresponding area.
```

Comment:
I guess this is auto-generated now?  Is it worth trying to figure out how to get that link back (i.e. put it in the right place in the python doc annotations)?

Reply:
Yeah, this will be auto-generated. I've added the HSA link to the field definitions